src/libs/SignatureChecker.sol

```diff
@@ -32,8 +32,27 @@ library SignatureChecker {
         }
     }
 
+    /**
+     * @dev    Decodes an ECDSA/secp256k1 short signature as defined by EIP2098
+     *         from a byte array to standard v, r, and s parameters.
+     * @param  signature A byte array ECDSA/secp256k1 short signature.
+     * @return r         An ECDSA/secp256k1 signature parameter.
+     * @return  vs     An ECDSA/secp256k1 short signature parameter.
+     */
```

Comment:
```suggestion
    /**
     * @dev    Decodes an ECDSA/secp256k1 short signature as defined by EIP2098
     *         from a byte array to standard v, r, and s parameters.
     * @param  signature A byte array ECDSA/secp256k1 short signature.
     * @return r         An ECDSA/secp256k1 signature parameter.
     * @return vs        An ECDSA/secp256k1 short signature parameter.
     */
```

Reply:
Fixed in this commit: https://github.com/MZero-Labs/common/pull/8/commits/fcb4ccb4fe10e6a5002111e1e3766c056b6bfbec